### PR TITLE
Underwater Rover

### PR DESCRIPTION
**Pull request by ayushkumardixit**

### CODE REVIEW

**Comment by agshubhs** [comment]
@ayushkumardixit  Its looking Nice only minute changes are required.
1. Order of subsection in the Readme file is not correct. *Please refer guidelines for order* 
2. Put the screeshots of the solidworks design rather than the image capture from camera in solidwork word.
3. Make a table for cost structure.

File: README.md [hunk 1]
```diff
@@ -1,2 +1,91 @@
...
+## Mechanical Aspect
+
+ Here main problem  is waterproofing of whole system specially at shaft of motors. Here we have used wax and vaseline for waterproofing.
+ And another problem is selection of proper motor producing enough torque to counterbalance drag. To produce sufficient thrust, here we have used DC motors of 300RPM.
+ 
+ ## Electronic aspect
+ 
+ The Arduino code acts as a 4-button controller with a switch, controlling 4 motors via 2 motor drivers. When the switch is in ON state, all the motors move in clockwise direction, thus enabling UP and FORWARD movement. When the switch is in OFF state, all the motors move in anticlockwise direction, thus enabling DOWN and BACKWARD movement. In both the switch's states, RIGHT and LEFT is possible. It drives each motor with the same PWM value signal of 240.
+ 
+ ![](https://github.com/ayushkumardixit/Underwater-Rover/blob/master/Mechanical%20Design/Solidworks%20screenshot.png)
```
Comment (agshubhs) [inline README.md]: Put the cropped screenshot consisting fo the design only and It should be inserted in mechanical aspect section not in Electronics Aspect

File: README.md [hunk 2]
```diff
@@ -1,2 +1,91 @@
...
+ 
+ ## Cost Structure 
+
+Mechanical parts:
```
Comment (agshubhs) [inline README.md]: Make a table for Cost Structure

**Comment by agshubhs** [comment]
@ayushkumardixit  Write the Limitations, ELectronics Aspect, and Mechanical Aspect in points rather than in paragraph

File: README.md [hunk 3]
```diff
@@ -1,2 +1,86 @@
...
+
+## Future Improvements
+
+1. It can be made wireless with help of devices like Bluetooth.
+2. By improving electronic system and mechanical strength, it can reach greater depths.
+
+## TEAM MEMBERS
+
+1.[Ayush Kumar Dixit](https://github.com/ayushkumardixit)
+
```
Comment (agshubhs) [inline README.md]: Remove this line space between the team members

File: README.md [hunk 4]
```diff
@@ -1,2 +1,86 @@
...
+4.[Rahul Panjwani](https://github.com/rahulpanjwani)
+
+
+
+
+## Mentor
+
+1.Abhishek Sehgel
+
```
Comment (agshubhs) [inline README.md]: Remove the line space from this section also

File: README.md [hunk 5]
```diff
@@ -1,2 +1,86 @@
...
+## Limitation
+
+This rover can not work beyond a certain depth, because of increase in pressure, water will begin to enter inside the body and also we have to check the frequency of the wave we are using should not match with frequency produced by vibration of motors and also should not  match with the resonace frequency.
+
+## Future Improvements
+
+1. It can be made wireless with help of devices like Bluetooth.
+2. By improving electronic system and mechanical strength, it can reach greater depths.
+
+## TEAM MEMBERS
```
Comment (agshubhs) [inline README.md]: Write the headings  normally. Don't write in capital letters.

**Comment by agshubhs** [comment]
@ayushkumardixit Do the following changes

**Comment by agshubhs** [comment]
@ayushkumardixit  
1. In readme file remove necessary spaces.
2. Number list syntax is wrong that's why you are not getting the list in Electronic Aspects and Mechanical Aspect. 
Correct them.

File: README.md [hunk 6]
```diff
@@ -42,47 +26,30 @@ Humans have been able to explore only about 20% of sea life. With the help of th
...
 ## Team Member
-1.[Ayush Kumar Dixit](https://github.com/ayushkumardixit)
-
-2.[Masih Ahmed](https://github.com/anon766)
-
-3.[Parth Gupta](https://github.com/parthg2211)
-
-4.[Rahul Panjwani](https://github.com/rahulpanjwani)
-
+1. [Ayush Kumar Dixit](https://github.com/ayushkumardixit)
```
Comment (agshubhs) [inline README.md]: Remove this extra space between the names and mentors

File: README.md [hunk 7]
```diff
@@ -1,2 +1,58 @@
...
+3. We have used 4 motors in which 2 of them provides vertical thrust and other 2 provides horizontally . with help of these we can rotate it in a horizontal plane and also we can travel vertically .
+![](https://github.com/ayushkumardixit/Underwater-Rover/blob/master/Mechanical%20Design/Solidworks%20screenshot.png)
+ ## Electronic aspect
+1. The Arduino code acts as a 4-button controller with a switch, controlling 4 motors via 2 motor drivers.
+2. When the switch is in ON state, all the motors move in clockwise direction. Thus enabling  UP and FORWARD movement. 
+3. When the switch is in OFF state, all the motors move in anticlockwise direction, thus enabling DOWN and BACKWARD movement. 
+4. In both the switch's states, RIGHT and LEFT is possible.
+5. It drives each motor with the same PWM value signal of 240
+ ## Cost Structure 
+ | Materials  | Cost |
```
Comment (agshubhs) [inline README.md]: @ayushkumardixit  
Write Cost(INR) Inplace of Cost

Comment (agshubhs) [reply README.md]: This change is needed @ayushkumardixit

**Comment by ayushkumardixit** [comment]
Is this done??

**Comment by agshubhs** [comment]
@ayushkumardixit write Cost(INR) in place of Cost in Cost Structure subsection.

**Comment by agshubhs** [comment]
@ayushkumardixit 
There are two nested Mechanical Design folder. Remove one of them if they are same.

**Comment by agshubhs** [comment]
@ayushkumardixit  See the mechanical design folder

**Comment by agshubhs** [comment]
Nice Work @ayushkumardixit

**Pull request merged**